samtranslator/model/sam_resources.py

```diff
@@ -2204,12 +2204,16 @@ def __init__(
     @cw_timer
     def to_cloudformation(self, **kwargs: Any) -> List[Resource]:
```

Comment:
I don't see any transform failure tests; is it expected that there are no new transform failure cases in the code?

Reply:
There have been transform error tests for authorisers. I haven't change them